src/Morphir/JsonSchema/AST.elm

```diff
@@ -16,10 +16,27 @@ type alias Schema =
     }
 
 
+type alias UniqueItems =
+    Bool
+
+
+type alias NoOfItems =
+    Int
+
+
 type SchemaType
     = Integer
-    | Array SchemaType
+    | Array ArrayType UniqueItems
     | String
     | Number
     | Boolean
     | Object (Dict String SchemaType)
+    | Const String
+    | Ref TypeName
+    | OneOf (List SchemaType)
+    | Null
+
+
+type ArrayType
+    = ListType SchemaType
+    | TupleType (List SchemaType) NoOfItems
```

Comment:
Please add docs to the AST. Especially to the places where we had to deviate from the JSON Schema spec.